docker-compose.override.yaml

```diff
@@ -7,9 +7,19 @@ services:
       target: development
     image: ffc-pay-file-receiver-development
     container_name: ffc-pay-file-receiver-development
-    ports:
-      - "3000:3000"
-      - "9229:9229"
     volumes:
       - ./app:/home/node/app
       - ./package.json:/home/node/package.json
+    environment:
+      AZURE_STORAGE_BLOB_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-file-receiver-azurite:10000/devstoreaccount1;
+      AZURE_STORAGE_SHARE_ACCOUNT_NAME: ${AZURE_STORAGE_ACCOUNT_NAME:-}${AZURE_STORAGE_SHARE_ACCOUNT_NAME}
+      AZURE_STORAGE_SHARE_CONNECTION_STRING: DefaultEndpointsProtocol=https;AccountName=${AZURE_STORAGE_SHARE_ACCOUNT_NAME};AccountKey=${AZURE_STORAGE_SHARE_ACCOUNT_KEY};EndpointSuffix=core.windows.net
+
+  ffc-pay-file-receiver-azurite:
+    volumes:
+      - azurite_data:/data
+    ports:
+      - "10073:10000"
```

Comment:
Could I be cheeky and ask for this to be changed to `10074`?  Just because another payment service we're working on uses `10073`